src/procedures/FinalizeSto.ts

```diff
@@ -41,6 +75,13 @@ export class FinalizeSto extends Procedure<FinalizeStoProcedureArgs> {
       });
     }
 
+    function throwStoModuleError() {
```

Comment:
Ideally we should avoid using nested functions unless there is a good reason for it. It's slightly preferable to assign the error to a variable and then reuse that